package.json

```diff
@@ -112,5 +112,8 @@
       }
     },
     "collectCoverage": true
+  },
+  "dependencies": {
+    "ajv": "^6.9.1"
```

Comment:
Seems suspect to me. Shouldn't it be needed

If i have missed something and it is needed please put in devDependencies and fix version number to "6.9.1"

Reply:
`npm` whined about "peer dependencies need to be manually installed" or some such. Does it not for you? I think it was in the context of eslint, but that's from memory.

Here are the dependencies:

```
$ docker-compose run web npm list ajv
openlibrary@1.0.0 /openlibrary
+-- ajv@6.9.1 
+-- css-loader@2.1.1
| `-- schema-utils@1.0.0
|   `-- ajv@6.9.1  deduped
+-- eslint@6.8.0
| +-- ajv@6.11.0 
| `-- table@5.4.6
|   `-- ajv@6.11.0  deduped
+-- less@3.10.3
| `-- request@2.88.0
|   `-- har-validator@5.1.0
|     `-- ajv@5.5.2 
+-- style-loader@1.1.2
| `-- schema-utils@2.6.4
|   `-- ajv@6.11.0 
+-- stylelint@9.5.0
| `-- table@4.0.3
|   `-- ajv@6.9.1  deduped
`-- webpack@4.41.2
  `-- ajv@6.10.2 
```

The remaining audit vulnerabilities are:

```
$ docker-compose run web npm audit 
                                                                                
                       === npm audit security report ===                        
                                                                                
# Run  npm install --save-dev jquery@3.4.1  to resolve 2 vulnerabilities
SEMVER WARNING: Recommended action is a potentially breaking change
                                                                                
  High            Cross-Site Scripting (XSS)                                    
                                                                                
  Package         jquery                                                        
                                                                                
  Dependency of   jquery [dev]                                                  
                                                                                
  Path            jquery                                                        
                                                                                
  More info       https://npmjs.com/advisories/328                              
                                                                                


                                                                                
  Moderate        Prototype Pollution                                           
                                                                                
  Package         jquery                                                        
                                                                                
  Dependency of   jquery [dev]                                                  
                                                                                
  Path            jquery                                                        
                                                                                
  More info       https://npmjs.com/advisories/796                              
                                                                                


# Run  npm install --save-dev stylelint@13.0.0  to resolve 1 vulnerability
SEMVER WARNING: Recommended action is a potentially breaking change
                                                                                
  Low             Regular Expression Denial of Service                          
                                                                                
  Package         braces                                                        
                                                                                
  Dependency of   stylelint [dev]                                               
                                                                                
  Path            stylelint > micromatch > braces                               
                                                                                
  More info       https://npmjs.com/advisories/786                              
                                                                                


# Run  npm update jquery --depth 2  to resolve 2 vulnerabilities
                                                                                
  High            Cross-Site Scripting (XSS)                                    
                                                                                
  Package         jquery                                                        
                                                                                
  Dependency of   slick-carousel [dev]                                          
                                                                                
  Path            slick-carousel > jquery                                       
                                                                                
  More info       https://npmjs.com/advisories/328                              
                                                                                


                                                                                
  Moderate        Prototype Pollution                                           
                                                                                
  Package         jquery                                                        
                                                                                
  Dependency of   slick-carousel [dev]                                          
                                                                                
  Path            slick-carousel > jquery                                       
                                                                                
  More info       https://npmjs.com/advisories/796                              
                                                                                


# Run  npm update stylelint --depth 2  to resolve 1 vulnerability
                                                                                
  Low             Regular Expression Denial of Service                          
                                                                                
  Package         braces                                                        
                                                                                
  Dependency of   stylelint-declaration-use-variable [dev]                      
                                                                                
  Path            stylelint-declaration-use-variable > stylelint > micromatch   
                  > braces                                                      
                                                                                
  More info       https://npmjs.com/advisories/786                              
                                                                                


                                                                                
                                 Manual Review                                  
             Some vulnerabilities require your attention to resolve             
                                                                                
          Visit https://go.npm.me/audit-guide for additional guidance           
                                                                                
                                                                                
  Low             Regular Expression Denial of Service                          
                                                                                
  Package         clean-css                                                     
                                                                                
  Patched in      >=4.1.11                                                      
                                                                                
  Dependency of   less-plugin-clean-css [dev]                                   
                                                                                
  Path            less-plugin-clean-css > clean-css                             
                                                                                
  More info       https://npmjs.com/advisories/785                              
                                                                                
found 7 vulnerabilities (3 low, 2 moderate, 2 high) in 886229 scanned packages
  run `npm audit fix` to fix 3 of them.
  3 vulnerabilities require semver-major dependency updates.
  1 vulnerability requires manual review. See the full report for details.
```

Reply:
Should we use "peerDependencies" package.json key then?

Reply:
I'll circle back and retest. If it was a warning, not an error, and is an optional dependency that we don't need, perhaps it can just be ignored.